vscode-lean4/src/utils/fsHelper.ts

```diff
@@ -0,0 +1,39 @@
+/* eslint-disable @typescript-eslint/no-unsafe-assignment */
+import { readFile, promises, PathLike } from 'fs';
+
+
+export async function fsExistHelper(pathFile: PathLike): Promise<boolean> {
```

Comment:
Ok, I think we can now give this a nicer name, how about "fileExists"

Reply:
Sure!!